azure-pipelines.yml

```diff
@@ -219,17 +219,17 @@ stages:
               Add-Content build/version.txt $(SONAR_PROJECT_VERSION)
             displayName: 'Write project version in file'
 
-
           - task: DownloadSecureFile@1
             displayName: 'Download Maven settings'
             name: mavenSettings
             inputs:
               secureFile: 'maven-settings.xml'
 
-          - powershell: .\scripts\generate-packages.ps1 -sourcesDirectory "$(Build.SourcesDirectory)"
+          - powershell: |
+              . .\scripts\generate-packages.ps1
+              Run
```

Comment:
To allow testing I move the main behavior to a new function `Run`

Reply:
Why did you not put it on a single line?

Reply:
Hmm, I don't know how :D

Reply:
I just realised, you are doing two things. First, sourcing the file, then running `Run`.
Ignore this comment, resolving.